src/template-tags/tickets.php

```diff
@@ -1428,4 +1421,4 @@ function tribe_tickets_is_enabled_post_context( $post = null ) {
 		 */
 		return apply_filters( 'tribe_tickets_is_enabled_post_context', false, $post_types, $context );
 	}
-}
+}
```

Comment:
:no_entry_sign: File must end with a newline character

<sub>hash: 54e1664c1d243f82e819937d62bc2769</sub>

Reply:
@borkweb I thought it best practice _not_ to have empty/new line at the end of PHP files?

Reply:
```suggestion
}

```

Not sure there but we can just fix that here